oneflow/core/functional/impl/nn_functor.cpp

```diff
@@ -2123,6 +2123,10 @@ class CtcLossFunctor {
                            const std::shared_ptr<one::Tensor>& target_lengths,
                            const int64_t& max_target_length, const int64_t& blank,
                            const bool& zero_infinity, const std::string& reduction) const {
+    const auto& input_lengths_flatten =
+        JUST(functional::Flatten(input_lengths, 0, input_lengths->shape()->NumAxes() - 1));
```

Comment:
```suggestion
        JUST(functional::Flatten(input_lengths, 0, input_lengths->ndim() - 1));
```
这里可以用 ndim 来代替 shape -> NumAxes
除此之外，第三个参数也可以用 -1 代替，可以看下 functional_api.yaml 里面 flatten 的函数签名
下同